inventories/nos_event_service/inventory.yml

```diff
@@ -9,6 +9,10 @@ nos_event_service:
     events_google_pubsub_credentials_json_path: '{{inventory_dir}}/group_vars/all/vault_pubsub-credentials.json'
     events_listen_address: 8008
     events_environment: PRODUCTION
+    events_pyroscope_application_name: events.nos.social
+    events_pyroscope_server_address: https://pyroscope.nos.pink
```

Comment:
Do we want to use a nos/planetary domain here?  If I remember correctly, nos.pink is a personal domain?  If I am misremembering, please ignore this comment!

Reply:
Yeah I quickly spin up a server, I just need this for a couple of days for debugging purposes. If we decide to leave it running then we can move it to a different domain, most likely `planetary.tools`.